python/libyang/data.py

```diff
@@ -152,17 +154,29 @@ def path(self):
         finally:
             lib.free(path)
 
-    def validate(self, data=False, config=False, strict=False, trusted=False,
-                 no_yanglib=False):
+    def validate(self, data=False, config=False, get=False, strict=False,
+                 trusted=False, no_yanglib=False):
         flags = parser_flags(
-            data=data, config=config, strict=strict, trusted=trusted,
+            data=data, config=config, get=get, strict=strict, trusted=trusted,
             no_yanglib=no_yanglib)
         node_p = ffi.new('struct lyd_node **')
         node_p[0] = self._node
         ret = lib.lyd_validate(node_p, flags, ffi.NULL)
         if ret != 0:
             self.context.error('validation failed')
 
+    def merge(self, source, destruct=False, no_siblings=False, explicit=False):
+        flags = 0
+        if destruct:
+            flags |= lib.LYD_OPT_DESTRUCT
+        if no_siblings:
+            flags |= lib.LYD_OPT_NOSIBLINGS
+        if explicit:
+            flags |= lib.LYD_OPT_EXPLICIT
```

Comment:
Can these flags be used for something else than `lyd_merge()`? If so, maybe you should wrap this in a function like `parser_flags()`. If not, it can stay like this.

Reply:
In fact, I looked at the C header, and these flags are into the "parser options". Can you handle them directly into `parse_flags()` along with the others? Of course, the boolean arguments can stay the same.

Reply:
Good suggestion. Done!